### DISCUSSION TITLE
How to avoid having JSON saved as a string?

### DISCUSSION THREAD

**Discussion by BPilot253** [pocketbase/pocketbase#2595] [Q&A] [answered]
So I recently updated to 0.16.3 and noticed that the way JSON saves from the admin UI has changed. It now keeps saving it as a string, e.g. I try to save the following JSON:
```
{
    "02-01": {
        driverId: 252179607,
        assignedBus: 209,
        notes: ""
    }
}
```
but it keeps getting saved as
```
{\"02-01\": {driverId: 252179607,assignedBus: 209,notes:\"\"}}
```
Is there a way to avoid this behaviour? It seems to save normally if done through the JS SDK. (although I tested on an older version of the SDK)

**Comment by danibjor** [comment]
Because it’s not valid json, so it saves it as a string. The keys in the child object needs to be wrapped in “driverId” ..

**Reply by BPilot253** [reply]
Yep, I completely forgot about that and came here to say I figured it out. Thanks anyway.

**Comment by BPilot253** [comment, marked as answer]
Nevermind... I forgot I needed to wrap the keys in parantheses..
